root/static/lib/leaflet/leaflet.markercluster-src.js

```diff
@@ -0,0 +1,2616 @@
+/*

```

Comment:
Let's add "leaflet.markercluster" to package.json instead of importing the code (we generally don't do that with JS libs anymore). Then in common.js, just do `require('leaflet.markercluster/dist/leaflet.markercluster-src')`.